### DISCUSSION TITLE
R-Link reseat talisman

### DISCUSSION THREAD

**Discussion by Marian25** [cedricp/ddt4all#863] [Q&A]
I have a problem with Renault Talisman, I upgraded from version 3 to version 9 and I had 100 errors, then I downgraded to version 2, which resets the system, the good part is that I fixed some problems, but the bad part is that version 2 it's for Scenic but I have Talisman, there are still problems, it can't connect to bluetooth, and I'm asking for help if anyone has a file with version 2 for Talisman,
P.S. I went to the dealer, but there they ask me for too much money, which I can't afford because I damaged my father's car

**Comment by AlMeu** [comment]
You messed up all system upgrading wrong.
Upgrade to v7.0 from 3.3 is doing only at dealer with special procedure and even like this the chance of success is about 50%.
I don't think you can fix anymore that unit and almost sure you need change the R-Link unit, as perhaps dealer told you also.
The problem is that all units in these cars are locked to work only in that car.
If you look for change the unit perhaps I can help, and you can leave an email for contact you.

**Comment by Furtif** [comment]
issue converted to discution
